scwx-qt/source/scwx/qt/main/check_privilege.cpp

```diff
@@ -0,0 +1,136 @@
+#include "scwx/qt/settings/general_settings.hpp"
```

Comment:
Should use `<>` instead of `""` for consistency with other files, since these are not relative paths.